rakefile.rb

```diff
@@ -53,7 +53,7 @@
 
 desc "Executes xUnit tests"
 xunit :xunit => [:compile] do |xunit|
-    tests = FileList["src/**/#{CONFIGURATION}/Nancy.Bootstrappers.StructureMap.Tests.dll"].exclude(/obj\//)
+    tests = FileList["test/**/#{CONFIGURATION}/Nancy.Bootstrappers.StructureMap.Tests.dll"].exclude(/obj\//)

```

Comment:
I think you need to add a `CLEAN.include(FileList["test/**/#{CONFIGURATION}"])` on line 25